lib/reporter.js

```diff
@@ -110,6 +110,17 @@ class AllureReporter extends events.EventEmitter {
             this.currentHooks.pop();
         })
 
+        this.on('runner:screenshot', (screenshot) => {
+
+            var filename = this.getScreenshotFilename(screenshot),
+                data = new Buffer(screenshot.data, 'base64')
+
+            this.print('screenshot', filename)
+            this.allure.addAttachment(filename, data)
```

Comment:
FYI, filename is not an actual filename. It is just a string to display this screenshot in attachment list.